### DISCUSSION TITLE
inaccurate trades number

### DISCUSSION THREAD

**Discussion by kfkf9999** [kernc/backtesting.py#264] [Q&A] [answered]
I test a simple strategy, like 
```py
if self.data.Close[-1] > self.data.Close[-2]:
    self.buy()
if self.data.Close[-1] < self.data.Close[-2]:
    self.sell()
```
however, when bt.plot(), Trades(1) showed.

What's happening?

**Comment by kernc** [comment, marked as answer]
[Buy and sell methods](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy), by default, make orders for size equivalent to full free equity (available cash + margin), which fluctuates in time depending on the underlying OHL**C** values.

You might want to consider passing explicit buy/sell `size=` parameter.

Alternatively, you should be able to control the explicit closing of trades with [`trade.close()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Trade.close) or [`position.close()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Position.close).

**Reply by kfkf9999** [reply]
Thank you! I resolve!

**Comment by samueltg92** [comment]
@kernc hello bro!, is the size parameter given in USD terms? I mean, when I see the _trades results I see "-883" (for example). Is that number the size in USD (the quantity of USD I use to open the trade)?

**Reply by kernc** [reply]
The `size=` parameter matches [`Order.size`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Order.size) / [`Trade.size`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Trade.size).
